engines/bastion_katello/app/assets/javascripts/bastion_katello/i18n/locale/ja.po

```diff
@@ -5066,8 +5066,8 @@ msgid "weekly"
 msgstr "毎週"
 
 #: app/assets/javascripts/bastion_katello/host-collections/details/views/host-collection-add-content-hosts.html
-msgid "{{ addContentHostsTable.numSelected }} Selected"
-msgstr "{{ addContentHostsTable.numSelected }} 件選択済み"
+msgid "{{ addHostsTable.numSelected }} Selected"
+msgstr "{{ addHostsTable.numSelected }} 件選択済み"
```

Comment:
Usually these shouldn't be updated manually.  However, if I don't update these .po files to point to the correct table then these values will be undefined in all languages but English.  This is because I updated the host collection add/list content hosts to use details nutupane (not reflected in these changes, i.e. it should be `detailsTable.numSelected`).